src/Writer/FilterPluginManager.php

```diff
@@ -1,61 +1,15 @@
 <?php
 /**
- * Zend Framework (http://framework.zend.com/)
+ * @author    Antoine Hedgecock <antoine.hedgecock@gmail.com>
  *
- * @link      http://github.com/zendframework/zf2 for the canonical source repository
- * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
- * @license   http://framework.zend.com/license/new-bsd New BSD License
+ * @copyright Interactive Solutions AB
  */
 
 namespace Zend\Log\Writer;
 
-use Zend\Log\Exception;
-use Zend\Log\Filter;
-use Zend\ServiceManager\AbstractPluginManager;
-
-class FilterPluginManager extends AbstractPluginManager
+/**
+ * @deprecated has simply be moved to the parent directory
```

Comment:
@macnibblet That's not my point. Moving them breaks backwards compatibility, and I'd like to reduce the number of such breakages. That said, your strategy does work for that, as it's having the originals extend the new ones. So, objection withdrawn. :smile: